File: Arrays & Strings/#1394 - Lucky integer in an Array - Easy/solution.py
class Solution:
    def findLucky(self, arr):
        temp = 0
        lucky = -1
        count = 0

        for i in range(len(arr)):
            count = 0
            temp = arr[i]

            for j in range(len(arr)):
                if arr[j] == temp:
                    count += 1

            if count == len(arr):
                lucky = temp

        return lucky

File: Arrays & Strings/#1394 - Lucky integer in an Array - Easy/solution.cpp
class Solution {
public:
    int findLucky(vector<int>& arr) {
        int temp = 0, lucky = -1, count = 0;

        for (int i = 0; i < arr.size(); i++) {
            count = 0;
            temp = arr[i];

            for (int j = 0; j < arr.size(); j++) {
                if (arr[j] == temp)
                    count++;
            }

            if (count == arr.size())
                lucky = temp;
        }

        return lucky;
    }
};
